### PR TITLE
Root element

### PR DESCRIPTION
**Pull request by billybonks**
Allow setting the root element, because the app is just a window in the dom. so helpers are picking up qunit ui.

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -9,12 +9,27 @@ class Config {
       toggle: [],
       button: [],
     }
+    this._rootElement = document;
```
Comment (brookjordan) [inline src/config.js]: `document` isn't an element.
Shouldn't this be `document.documentElement` at least?

Comment (billybonks) [reply src/config.js]: good point, this does work though. can raise a pr?

File: src/config.js [hunk 2]
```diff
@@ -9,12 +9,27 @@ class Config {
...
+
+  get rootElement(){
+    return this._rootElement;
+  }
+
+  set rootElement(element){
+    if(element) {
+      this._rootElement = element;
+    } else {
+      throw new Error('You tried to set root element to null');
```
Comment (brookjordan) [inline src/config.js]: Maybe:
```js
if(element instanceof Element) {
  this._rootElement = element;
} else {
  throw new Error(`rootElement Error: You tried to set root element to ${element}`);
}
```

File: src/config.js (hunk 1 above)
Comment (brookjordan) [reply src/config.js]: Sure.
It'll stop working if you ever try to use the `rootElement` as an element.

File: src/config.js (hunk 2 above)
Comment (billybonks) [reply src/config.js]: 👍

**Pull request merged**